components/service/glean/README.md

```diff
@@ -1,17 +1,59 @@
 # [Android Components](../../../README.md) > Service > Glean
 
-A client-side telemetry SDK for collecting metrics and sending them to Mozilla's telemetry service (eventually replacing [service-telemetry](../telemetry/README.md)).
+A client-side telemetry SDK for collecting metrics and sending them to Mozilla's telemetry service
+(eventually replacing [service-telemetry](../telemetry/README.md)).
 
 ## Usage
 
 ### Setting up the dependency
 
-Use Gradle to download the library from [maven.mozilla.org](https://maven.mozilla.org/) ([Setup repository](../../../README.md#maven-repository)):
+Use Gradle to download the library from [maven.mozilla.org](https://maven.mozilla.org/) 
+([Setup repository](../../../README.md#maven-repository)):
 
 ```Groovy
 implementation "org.mozilla.components:service-glean:{latest-version}"
 ```
 
+### Integrating with the build system
+
+In order for Glean to work, a Python environment must be accessible at build time. This is done
+automatically by the [`com.jetbrains.python.envs`](https://github.com/JetBrains/gradle-python-envs/)
+plugin. The plugin **must** be manually enabled by adding the following `plugins` block at the top
+of the main `build.gradle` file.
+
+```Groovy
+plugins {
+    id "com.jetbrains.python.envs" version "0.0.25"
+}
+```
+
+Right before the end of the same file, the glean build script must be included, as shown below:
+
+```Groovy
+apply from: 'https://github.com/mozilla-mobile/android-components/raw/master/components/service/glean/scripts/sdk_generator.gradle'
```

Comment:
Do we want to specify a specific tag in this URL?  Problems could occur if this doesn't match the version of `service-glean` specified above.  I think that (for now) means we'll have to track along with `android-components` versions...

Reply:
Great point! Thanks for spotting this. I've updated the docs, can you check if the current phrasing makes any sense?